src/main/java/org/mule/extension/email/internal/mailbox/pop3/POP3PollingSource.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.internal.mailbox.pop3;
+
+import org.mule.extension.email.api.predicate.BaseEmailPredicateBuilder;
+import org.mule.extension.email.api.predicate.IMAPEmailPredicateBuilder;
+import org.mule.extension.email.api.predicate.POP3EmailPredicateBuilder;
+import org.mule.extension.email.internal.mailbox.BaseMailboxPollingSource;
+import org.mule.runtime.extension.api.annotation.Alias;
+import org.mule.runtime.extension.api.annotation.param.MediaType;
+import org.mule.runtime.extension.api.annotation.param.Optional;
+import org.mule.runtime.extension.api.annotation.param.Parameter;
+import org.mule.runtime.extension.api.annotation.param.display.DisplayName;
+
+import javax.mail.Folder;
+import javax.mail.Message;
+
+import static java.util.Optional.ofNullable;
+
+/**
+ * Retrieves all the emails from an POP3 mailbox folder.
+ *
+ * @since 1.1
+ */
+@DisplayName("On Email - POP3")
```

Comment:
same here

Reply:
but not always are new, this case probably most of the emails are not new